### PR TITLE
dhaas pre-release feedback

### PR DESCRIPTION
**Pull request by thisisdhaas**
(see comments throughout the diff)

### CODE REVIEW

File: src/introduction.md [hunk 1]
```diff
@@ -14,9 +14,9 @@ While it took B12 about a week to ship the first copy generation feature in our
...
+For lack of a shared understanding of how to best communicate about AI-powered tools, the best we can offer is transparency into how we do it ourselves. To that end, we're releasing *How we AI*, a handbook that explains how B12 understands and communicates AI in our product. We're licensing *How we AI* under a [permissive CC-BY-SA-4.0 license](https://github.com/b12io/how-we-ai/blob/main/LICENSE.md) so that other organizations can use any element they want while making changes that match their needs. We hope that releasing *How we AI* opens B12 to constructive critique while enabling a conversation around the best way to build and talk about AI-enabled products.
```
Comment (thisisdhaas) [inline src/introduction.md:17]: (removed self-link which is bad for SEO)

File: src/introduction.md [hunk 2]
```diff
@@ -14,9 +14,9 @@ While it took B12 about a week to ship the first copy generation feature in our
...
 
-*How we AI* is a work in progress. Today's edition speaks to:
+*How we AI* is a work in progress. So far, we've addressed:
```
Comment (thisisdhaas) [inline src/introduction.md:19]: "today's edition" felt awkward, esp as we update it over time.

File: src/limitations-examples.md [hunk 3]
```diff
@@ -5,23 +5,37 @@ Before releasing generative AI functionality in our product, we follow our [eval
...
-* **Plagiarism**. We did not find examples of plagiarism. In 7/40 examples, [Grammarly's plagiarism detector](https://www.grammarly.com/plagiarism-checker) identified 6-15% document similarity to other content on the web. Upon inspection, every case of similarity involved common idioms or platitudes (e.g., the phase `keen eye for detail` in a fictional team member bio).
-* **Repetition**. In 3/40 examples, we identified a repetitive word or phrase, and in 1/40 examples, we identified an unnecessary paragraph.
-* **Grammatical opportunities**. In 7/40 examples, we identified some grammatical error or opportunity. While only one of these was an outright grammatical mistake, we commonly saw opportunities to use a more active voice in the copy.
-* **Staying on topic**. In 5/40 examples, we identified an opportunity for the text to be more on-topic, and in 4/40 examples, we identified opportunities for the text to make a stronger argument for the prompt.
+### Bias and length discrepancies
```
Comment (thisisdhaas) [inline src/limitations-examples.md:8]: ugh the diff sucks:
* Re-formatted to make the types of issue top-level headers and reduce unnecessary bullet nesting.
* Tweaked text in a parenthetical under accuracy (old line 23 vs. new line 29)
* Added a sentence speculating on higher risk of plagiarism in longer form text outputs (new line 32)

File: src/limitations.md
Comment (thisisdhaas) [inline src/limitations.md:1]: (this file looked empty, just cleaning up)

File: src/limitations-examples.md [hunk 4]
```diff
@@ -5,23 +5,37 @@ Before releasing generative AI functionality in our product, we follow our [eval
...
...e "facts" would serve to prompt a user to consider their own promotional nuances, but exemplify why a careful review of the machine-generated text is necessary. A limitation of our evaluation is that many of our examples involved blog outlines (factual inaccuracies are more likely to be in the blog body than the outline) and team member bios (for fictional team members), for which it was difficult to evaluate factual accuracy. The two prompts were `Send a free trial offer as part of our ongoing promotion.` and `Send a free trial offer as part of our black history month promotion for black-owned businesses.`. The made-up facts were `We offer powerful outreach tools that can boost your customer conversion rates by up to 20%.` and `With {product name}, you can streamline your processes and handle more transactions with less effort. We also provide powerful analytics tools to help you understand your customer base better, as well as personalized marketing campaigns to reach out to new customers.` In some emails it used placeholders in brackets like `{product name}` to identify placeholder content but in the 2 cases mentioned above it just made up facts without putting them in brackets.
```
Comment (marcua) [inline src/limitations-examples.md]:
```suggestion
Made-up facts appeared in 2/40 examples, both of them around an email announcing a free trial. In both cases, we believe the "facts" would serve to prompt a user to consider their own promotional nuances, but exemplify why a careful review of the machine-generated text is necessary. A limitation of our evaluation is that many of our examples involved blog outlines (factual inaccuracies are more likely to be in the blog body than the outline) and team member bios (for fictional team members), for which it was difficult to evaluate factual accuracy. The two prompts were `Send a free trial offer as part of our ongoing promotion.` and `Send a free trial offer as part of our black history month promotion for black-owned businesses.`. The made-up facts were `We offer powerful outreach tools that can boost your customer conversion rates by up to 20%.` and `With {product name}, you can streamline your processes and handle more transactions with less effort. We also provide powerful analytics tools to help you understand your customer base better, as well as personalized marketing campaigns to reach out to new customers.` In some emails, the generated text used placeholders in brackets like `{product name}` to identify placeholder content, but in the 2 cases mentioned above, facts were presented without putting them in brackets.
```

File: src/limitations-examples.md [hunk 5]
```diff
@@ -5,23 +5,37 @@ Before releasing generative AI functionality in our product, we follow our [eval
...
+
+### Plagiarism
+We did not find examples of plagiarism. In 7/40 examples, [Grammarly's plagiarism detector](https://www.grammarly.com/plagiarism-checker) identified 6-15% document similarity to other content on the web. Upon inspection, every case of similarity involved common idioms or platitudes (e.g., the phase `keen eye for detail` in a fictional team member bio). This might be resultant from the fact that we evaluated relatively short-form content (in the hundreds of characters): longer-form AI-generated text would offer more opportunities for plagiarism.
```
Comment (marcua) [inline src/limitations-examples.md]:
```suggestion
We did not find examples of plagiarism. In 7/40 examples, [Grammarly's plagiarism detector](https://www.grammarly.com/plagiarism-checker) identified 6-15% document similarity to other content on the web. Upon inspection, every case of similarity involved common idioms or platitudes (e.g., the phase `keen eye for detail` in a fictional team member bio). The lack of apparent plagiarism might result from the fact that we evaluated relatively short-form content (in the hundreds of characters): longer-form AI-generated text would offer more opportunities for plagiarism.
```

**Pull request merged**